VocalRaidAssistant.lua

```diff
@@ -127,11 +126,18 @@ function VRA:ChangeProfile()
 	self.profile = self.db.profile
 end
 
-function VRA:ChatCommand()
-	if self.ACD.OpenFrames["VocalRaidAssistantConfig"] then
-		self.ACD:Close("VocalRaidAssistantConfig")
-	else
-		self.ACD:Open("VocalRaidAssistantConfig")
+function VRA:ChatCommand(msg)
+	if (msg == "") then
```

Comment:
Will ChatCommand() register as hitting msg == "" or should it check for msg nil?

Reply:
It is not documented in AceConsole, but type(msg) = (empty) string, not nil.